examples/docs_beta_snippets/docs_beta_snippets/guides/components/migrating-definitions/2-definitions-before.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+import dagster_components as dg_components
```

Comment:
hmmm we should decide if we want to standardize on this alias. maybe worth a discussion in #project-components